semantic/src/site/globals/site.variables

```diff
@@ -164,3 +164,7 @@
 
 // Box shadow -- used in Login form
 @basicBoxShadow: rgb(204, 201, 201) 0px -3px 10px 0px;
+
+// Box shadow -- used in Dashboard 
+@templateBoxShadowColor: #d4d4d5;
+@templateBoxShadow: 0 1px 3px 0 @templateBoxShadowColor, 0 0 0 1px @templateBoxShadowColor;
```

Comment:
Nice! Can you move this color `@templateBoxShadowColor` up to be among other colous definitions? I think it can also be used on the `@basicBoxShadow` definition. it's pretty much the same.

Also - not related, but since we are discussing colors here. Can you add the latest colour from Debbie's design `#F0F0F0` to replace the color used for Application background?

Reply:
0370ff680b3a11ec8b080b32272bf61ea2c8d68e